standby/src/lib.rs

```diff
@@ -1021,7 +994,7 @@ impl Standby {
     ///
     /// Returns whether the bystander is fulfilled; if the bystander has been
     /// fulfilled then the channel is now closed.
-    #[cfg_attr(feature = "tracing", tracing::instrument(level = "trace"))]
+    #[tracing::instrument(level = "trace")]
```

Comment:
Only `trace` level events are recorded inside, so there's no need for the instrument to be at the trace level.
```suggestion
    #[tracing::instrument]